gen/flexlm_iptables

```diff
@@ -22,16 +22,16 @@ my $service_file_name = "$DIRECTORY/$::SERVICE_NAME";
 ####### output file ######################
 open SERVICE_FILE,">$service_file_name" or die "Cannot open $service_file_name: $! \n";
 
-my @resourcesData = $data->getChildElements;
-foreach my $rData (@resourcesData) {
-	my @membersData = $rData->getChildElements;
-	foreach my $mData (@membersData) {
-		my %memberAttributes = attributesToHash $mData->getAttributes;
-		$allowedIps{$_} = 1 foreach @{$memberAttributes{$A_U_IPS}};
+foreach my $resourceId ($data->getResourceIds()) {
```

Comment:
If I understand this script correctly, it prints unique list of IP addresses taken from all users assigned to the service/facility. Then we can iterate over all members directly, using `getMemberIdsForFacility()`. Also can you add explanatory comment, what is printed to the output file. Code is a little hard to read and understand.

Reply:
Done